bears/general/InvalidLinkBear.py

```diff
@@ -40,7 +40,10 @@ def find_links_in_file(file, timeout):
 
     def run(self, filename, file, timeout: int=DEFAULT_TIMEOUT):
         '''
-        Yields results for all invalid links in a file.
+        Find links in any text file and check if they are valid.
+
+        The bear will check if the page exists and propose an automatic fix if
+        it is a redirect.
```

Comment:
Add "If the page is not found or its a server side error, then those links will be displayed too"